steam/client/__init__.py

```diff
@@ -115,6 +118,13 @@ def _bootstrap_cm_list_from_file(self):
             self.cm_servers.merge_list(data['servers'])
             self._cm_servers_timestamp = int(data['timestamp'])
 
+    def _handle_server_list(self, msg):
```

Comment:
`self.servers` variable is not initialized here, so not only will be shared across class instances, it will have duplicate entries once the a single instance reconnects, as the loop below only appends